apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-help.tsx

```diff
@@ -36,7 +36,7 @@ const SamlIDPWizardHelp: FunctionComponent<Props> = (props: Props): ReactElement
             <Heading as="h5">Service provider entity ID</Heading>
             <p>
                 This value will be used as the <Code>&lt;saml2:Issuer&gt;</Code> in the SAML requests initiated from
-                { config.ui.productName } to external Identity Provider (IdP). You need to provide a unique value
+                {" "}{ config.ui.productName } to external Identity Provider (IdP). You need to provide a unique value
                 as the service provider entity id.
```

Comment:
```suggestion
                as the service provider entity ID.
```

Reply:
Thanks! Addressed this in commit 8bce665